docs/UserGuide.md

```diff
@@ -432,14 +432,20 @@ You should not need to read this, unless you run into some trouble.
 
 ### Detail: Bypassing validation
 
-Several fields have some validation rules (for example, you cannot use `ABCD` as a course code, since it does not conform to NUS course code format). Nevertheless, if you enter such a course code as input, the program will allow you to bypass the validation by adding `f/` to the end of the command.
+Several fields have some validation rules (for example, you cannot have email address with parentheses as there are no emails with parentheses).
+However, sometimes you might want to have the input that doesn't strictly adhere to these rules. For example, you may want to input both official and personal email addresses for a single person in an email field, like `E11111111@u.nus.com(school), helloworld@gmail.com(personal)`.
+To accommodate such cases, the program allows users to bypass the validation rules by appending `f/` to the end of their input. So, if you wanted to change 7th person's email address to the above email addresses despite the parentheses, you could do so by `edit 7 e/E11111111@u.nus.com(school), helloworld@gmail.com(personal) f/`.
```

Comment:
I feel it isn't really a good idea to include it in the "technical details" section. It's stated above "you should not need to read this unless you run into some trouble" after all.

Reply:
But I think if users consider using it and don't know how and why to use it, they are already in trouble.
Also, this is the only place to explain bypass deeply as the above section is not proper to have a wordy explanation.
![image](https://github.com/AY2324S2-CS2103T-T11-2/tp/assets/122252489/b9248d23-a1ab-45d2-8bbf-151541b404cf)